tests/unit/providers/test_anthropic_provider.py

```
@@ -1326,3 +1326,62 @@ def test_convert_models_list_mixed_created_at() -> None:
 
     assert [m.id for m in result] == ["with", "without"]
     assert [m.created for m in result] == [int(created_at.timestamp()), 0]
+
+
+def _anthropic_message(**extra: Any) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**extra`

(ANN401)